src/helpers/node.ts

```diff
@@ -39,6 +40,15 @@ export async function resolveDependencyPath(packageName: string): Promise<string
   }
 }
 
+/**
+ * Get the atom home which includes `packages` folder.
+ * Test runners such as `atom-jasmine3-runner` change this folder in the test env,
+ * so we need to install packages in the correct path
+ */
+function getAtomHomePath() {
+  return (atom as AtomEnvironment | null)?.getConfigDirPath() ?? process.env.ATOM_HOME ?? path.join(os.homedir(), '.atom')
```

Comment:
```suggestion
  return typeof atom === 'undefined' ? process.env.ATOM_HOME ?? path.join(os.homedir(), '.atom') : atom..getConfigDirPath()
```

Reply:
The code U wrote is the same. But Typescript handles it in the background.

Reply:
OK, I fixed this.

https://github.com/steelbrain/package-deps/pull/335/commits/74d6e727a366e4f8ba7adafdc62a429ddc9a7dc4